dhis-2/dhis-web-api/src/main/resources/openapi/TrackedEntitiesExportController.md

```diff
@@ -126,13 +181,24 @@ Get tracked entities with given UID(s).
 
 ### `*.parameter.TrackedEntityRequestParams.assignedUserMode`
 
+Get tracked entities and enrollments based on the user assignment in the events of these
+enrollments.
+
+- When `assignedUserMode=ME` tracked entities and enrollments that has at least one event assigned
```

Comment:
I don't think this mode exists https://github.com/dhis2/dhis2-core/blob/master/dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/AssignedUserSelectionMode.java